templates/includes/records/hierarchy-local.html

```diff
@@ -4,32 +4,33 @@
     <div class="container">
         <div class="row">
             <div class="col-4 col-md-2">
-                {% if page.previous_record %}
+                {% if page.previous_record.has_reference_number %}
                     <p class="hierarchy-local__nav-item">
                         <span class="hierarchy-local__nav-indicator hierarchy-local__nav-indicator--left"></span>
                         <span class="hierarchy-local__nav-label">{{ 'previous_record'|as_label }}</span>
-                        <a href="{% record_url page.previous_record %}" data-link="Previous" class="link--bold"><span class="sr-only">{{ 'previous_record'|as_label }}</span> {{ page.previous_record.iaid }}</a>
+                        <a href="{% record_url page.previous_record %}" data-link="Previous" class="link--bold"><span class="sr-only">{{ 'previous_record'|as_label }}</span> {{ page.previous_record.reference_number }}</a>
```

Comment:
> @TNA-Allan I have been told otherwise... we only want to show the links when there is a reference number, as we want what is displayed to be consistent.

The underlying `href url` shows `iaid` from `record_url`, but in [previous version before using record_url](https://github.com/nationalarchives/ds-wagtail/blob/566d39a2ec8bd3543eda73a7f36c117dd8649a78/templates/includes/records/hierarchy-local.html) of this template it shows underlying `href url` as `reference number` when it has a `reference number`.

Reply:
The link may work regardless, but we want the link text to always be a reference number... so that users always know what is being shown.